### ISSUE TITLE
hero mdbook, collection names interchanged

### ISSUE THREAD

**Issue by mik-tf** [crystaluniverse/crystallib#401]
## Situation

With hero mdbook, files from one collection **documentation** are all changed to another collection **intro**

e.g. _documentation/.../dashboard.md_ becomes _intro/dashboard.md_
image

## Screenshots

![image](https://github.com/freeflowuniverse/crystallib/assets/77026219/22dc361a-7ad0-4cb4-8963-c7f0e992a623)

![image](https://github.com/freeflowuniverse/crystallib/assets/77026219/d68bce20-4eff-4e1c-ade4-18180ab9b3b1)

![image](https://github.com/freeflowuniverse/crystallib/assets/77026219/a9ddc588-24c9-4764-bf3a-f7c357d5de9b)

## book_collections

```
  !!book.generate name:'tfmanual' title:'ThreeFold Manual Hero Version'
          url:'https://git.ourworld.tf/tfgrid/info_tfgrid/src/branch/main/books/tfmanual'
  
  !!doctree.add
      url:'https://git.ourworld.tf/tfgrid/info_tfgrid/src/branch/main/collections/tfmanual_intro'
  
  !!doctree.add
      url:'https://git.ourworld.tf/tfgrid/info_tfgrid/src/branch/main/collections/documentation'
  
```

**Comment by mik-tf** [comment]
Fix in #426

**Comment by mik-tf** [comment]
We have a workable fix in a recent PR
